gramps/gui/widgets/styledtexteditor.py

```diff
@@ -319,7 +319,8 @@ def make_tooltip_from_link(self, link_tag):
             if url.startswith("gramps://"):
                 obj_class, prop, value = url[9:].split("/")
                 display = simple_access.display(obj_class, prop, value) or url
-        return display
+        return display + \
```

Comment:
Please use implied line continuation inside parentheses instead of a backslash.  See the [Maximum Line Length](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) section in the PEP8 style guide.

Reply:
I guess the Eclipse/Pydev PEP8 checker doesn't check for that.  Usually if I just get rid of the warning flag, it passes pylint etc.